src/functions/getRegeneratedEmails/framework/handler.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayEvent, Context } from 'aws-lambda';
+
+import { bootstrapConfig } from '../../../common/framework/config/config';
+import Response from '../../../common/application/api/Response';
+import { getRegeneratedEmails } from './repositories/get-regenerated-emails-repository';
+import { HttpStatus } from '../../../common/application/api/HttpStatus';
+import createResponse from '../../../common/application/utils/createResponse';
+import * as joi from 'joi';
+import { gzipSync } from 'zlib';
+import { RegeneratedEmailsRecord } from '../../../common/domain/regenerated-emails';
+
+export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<Response> {
+  await bootstrapConfig();
```

Comment:
As per the issue we had other day in ref data service, we should be putting this inside the try catch to be handled properly.